tests/TestCase/Routing/RouterTest.php

```diff
@@ -1199,10 +1195,6 @@ public function testNamedRouteExists()
             ['action' => 'view'],
             ['_name' => 'Articles::view']
         );
-        
-        $this->assertTrue(Router::nameExists('users-index'));
-        $this->assertTrue(Router::nameExists('test'));
-        $this->assertTrue(Router::nameExists('Articles::view'));
```

Comment:
This test only connects a few routes and makes no assertions, seems like it isn't worth keeping around.